cookbooks/bcpc/templates/default/haproxy.cfg.erb

```diff
@@ -16,7 +16,6 @@ defaults
   maxconn  8000
   source <%="#{node[:bcpc][:management][:vip]}"%>
   mode  http
-  option httpclose
```

Comment:
Should we disable this only on the frontend(s)/backend(s) that we want keepalive on, or is this something we want to globally disable?